api/webhook/taskcluster_test.go

```diff
@@ -94,3 +97,29 @@ func TestVerifySignature(t *testing.T) {
 	assert.False(t, verifySignature(
 		message, "875a5feef4cde4265d6d5d21c304d755903ccb6", secret))
 }
+
+func TestCreateAllRuns_success(t *testing.T) {
+	requested := 0
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		requested++
+		w.Write([]byte("OK"))
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	err := createAllRuns(logrus.New(), nil, server.URL, "username", "password",
+		map[string][]string{"chrome": []string{"1"}, "firefox": []string{"1", "2"}},
+	)
+	assert.Nil(t, err)
+	assert.Equal(t, 2, requested)
+}
+
+func TestCreateAllRuns_error(t *testing.T) {
+	server := httptest.NewServer(http.NotFoundHandler())
```

Comment:
It would be nice to have a test that arranges for a specific request to error-out.

Reply:
Done. Now there are two test cases.